experimental/javascript_nodejs/qnamaker-activelearning-bot/Helpers/dialogHelper.js

```diff
@@ -0,0 +1,174 @@
+const { CardFactory } = require('botbuilder');
```

Comment:
please add the standard copyright header to this file:
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.